cmd/apply/cmdapply.go

```diff
@@ -97,9 +101,36 @@ func (r *ApplyRunner) RunE(cmd *cobra.Command, args []string) error {
 		emitStatusEvents = true
 	}
 
-	// Run the Applier. It will return a channel where we can receive updates
+	_, err = common.DemandOneDirectory(args)
```

Comment:
We should eventually clean this up, since we're just ignoring the return value.

Reply:
Done